feature-libs/pickup-in-store/components/presentational/pickup-options/pickup-options.component.ts

```diff
@@ -5,27 +5,40 @@
  */
 
 import {
+  AfterViewInit,
+  ChangeDetectorRef,
   Component,
   ElementRef,
   EventEmitter,
   inject,
   Input,
   OnChanges,
+  OnDestroy,
   Output,
   ViewChild,
+  TemplateRef,
 } from '@angular/core';
 import { FormControl, FormGroup } from '@angular/forms';
 import { FeatureConfigService, useFeatureStyles } from '@spartacus/core';
 import { PickupOption } from '@spartacus/pickup-in-store/root';
+import { TAB_MODE, Tab, TabComponent, TabConfig } from '@spartacus/storefront';
+import { Subscription, take } from 'rxjs';
 
+export enum PickupOptionsTabs {
```

Comment:
Could we move this to a `model.ts` file for easier import?

Reply:
Done